pages/faq/index.tsx

```diff
@@ -212,13 +211,13 @@ const FaqPage: NextPage = () => {
                   label={section}
                   onClick={() => handleChipClick(section)}
                   clickable
-                  color={selectedSection === section ? "primary" : "default"}
```

Comment:
Why is the selectedSection removed here and in after?

Reply:
It was used to not highlight the selected chip